docs/cloudformation.json

```diff
@@ -488,10 +489,17 @@
                 "Name": "EMPIRE_ECS_SERVICE_ROLE",
                 "Value": { "Ref": "ServiceRole" }
               },
-
               {
                 "Name": "EMPIRE_EC2_SUBNETS_PUBLIC",
                 "Value": { "Fn::Join": [ ",", [{ "Ref": "PubSubnetAz1" }, { "Ref": "PubSubnetAz2" }] ] }
+              },
+              {
+                "Name": "EMPIRE_EVENTS_BACKEND",
+                "Value": "sns"
+              },
+              {
+                "Name": "EMPIRE_SNS_TOPIC",
```

Comment:
There should probably be a condition for this - requiring that folks create an events topic is a bit annoying.